docs/tutorials/time-series.md

```diff
@@ -0,0 +1,738 @@
+requires recomputing the `daily_max` query for the corresponding date.
+In contrast, a discarded record is conceptually still part of the relation;
+however it will not participate in computing any future incremental updates
+and therefore does not need to be stored.
+
+:::warning
+
+Feldera does not automatically garbage collect [materialized tables and views](/sql/materialized).
+When a table or view is declared as materialized, it indicates an explicit user
+request to store its entire contents, making it accessible for
+[ad hoc queries](/sql/materialized/#ad-hoc-queries).
+However, if the materialized table contains an unbounded time series, it will
+continue to consume storage without limit, even if the associated views only
+require storing a bounded portion of the time series.
+
+:::
+
+
+## Emitting final values of a view with `emit_final`
+
+:::warning
+
+The `emit_final` feature is still experimental, and it may be removed
+or substantially modified in the future.
+
+:::
+
+By default, Feldera updates SQL views incrementally whenever new
+inputs arrive. Such incremental updates include deleting existing
+rows and inserting new rows into the view.  By monitoring these
+updates, the user can observe the most up-to-date results of the
+computation.  However, some applications only need to observe its
+final outputs, i.e., rows that are guaranteed to
+never get deleted or updated.
+
+Consider the following query that computes the sum of all purchases
+for each day.
+
+```sql
+CREATE VIEW daily_total AS
+SELECT
+    TIMESTAMP_TRUNC(ts, DAY) as d,
+    SUM(amount) AS total
+FROM
+    purchase
+GROUP BY
+    TIMESTAMP_TRUNC(ts, DAY);
+```
+
+Suppose that a new `purchase` record arrives every second.  In response,
+Feldera updates the `daily_total` value for the current date,
+deleting the old value and inserting a new one.  The user may observe
+86400 such updates per day.  In some cases this is precisely what the user
+wants--to keep track of the latest value of the aggregate with low latency.
+But if the application only requires the final value of the aggregate at
+the end of the day, handling all the intermediate updates may introduce
+unnecessary overheads. Some applications may not be able to handle
+intermediate updates at all.  Typical reasons for this are:
+
+* **The application is unable to process row deletions**. Some
+    databases and database connectors cannot ingest deletions
+    efficiently or at all.
+
+* **The application cannot handle a large volume of incremental
+    updates.**  As we see in this example, the volume of intermediate
+    updates can be much larger than the final output, potentially
+    overwhelming the application.
+
+* **The application is only allowed to act on the final
+    output**. Consider an application that raises an alarm, sends an
+    email, or performs some other irreversible action based on the
+    output or the view. Such actions should only be taken when the
```

Comment:
it's not obvious to me that alarms fall into this category; on the contrary. You should find more pertinent examples or just skip them.

You can point to the IRS example in the blog post